docker/dockerfile.tf

```diff
@@ -21,9 +21,13 @@ RUN pip install tensorflow-gpu==2.9.2 transformers==4.23.1\
 
 # DLFW Tensorflow packages
 COPY --chown=1000:1000 --from=dlfw /usr/local/lib/python3.8/dist-packages/tensorflow /usr/local/lib/python3.8/dist-packages/tensorflow/
+COPY --chown=1000:1000 --from=dlfw /usr/local/lib/python3.8/dist-packages/tensorflow-2.9.1+nv22.08.dist-info /usr/local/lib/python3.8/dist-packages/tensorflow-2.9.1+nv22.08.dist-info/
```

Comment:
It seems like you can't use ARG variables in a copy source or destination. So can't use the `TENSORFLOW_VERSION` arg in place of `22.08` here.

We can use a wildcard in the source `tensorflow-*.dist-info` , but not for the destination, because docker doesn't copy the directory folder name if you specify the destination as `dist-packages/` for example.